doc/security_need_to_know.md

```diff
@@ -0,0 +1,33 @@
+# About security issues
+
+This document aims to explain the process of identifying a security issue and the steps for managing a security issue.
+
+## identifying a security issue
+
+It is commonly stated that a security issue is an issue that:
+
+- Exposes sensitive information to unauthorized parties.
+- Allows unauthorized modification of data or system state.
+- Affects the availability of the system or its services.
+- Permits unauthorized access to the system.
+- Enables users to perform actions they should not be able to.
+- Allows users to deny actions they have performed.
+
+Given that WASI is a set of Capability-based APIs, all unauthorized actions are not supposed to happen. Most of the above security concerns can be alleviated. What remains for us is to ensure that Wasm modules' execution is secure. In other words, do not compromise the sandbox. Unless it is explicitly disabled beforehand.
```

Comment:
Maybe it can be rephrased as "What remains for us is to ensure that the execution of Wasm modules is secure."